data_processing/cws_power.ipynb

```diff
@@ -0,0 +1,182 @@
+    "\n",
+    "$Q = mc_p\\Delta T$ [J]\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def Q (m, c, delta_T):\n",
+    "    \"\"\"\n",
+    "    This function calculates the energy stored in joules. \n",
+    "    \n",
+    "    Parameters:\n",
+    "    -----------\n",
+    "    m : float\n",
+    "        This is the mass of the heated/cooled substance in [kg]\n",
+    "    c : float\n",
+    "        This is the specific heat capacity of the substance in \n",
+    "        [J/kg C]\n",
+    "    delta_T : float\n",
+    "        This is the change in temperature of the substance. \n",
+    "    \n",
+    "    Returns:\n",
+    "    --------\n",
+    "    q : float\n",
+    "        The energy stored in the substance. \n",
+    "    \"\"\"\n",
+    "    q = m*c*delta_T\n",
+    "    return q\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The energy stored in the full CWS tank is 935072326893.9393 joules\n"
+     ]
+    }
+   ],
+   "source": [
+    "tank_vol = 6.5e6 # gallons\n",
+    "\n",
+    "# convert gallons to cubic meters\n",
+    "tank_vol = tank_vol/264\n",
+    "\n",
+    "rho_w = 997 # kg/m^3\n",
+    "cp_w = 4186 # j/kg C\n",
+    "dT = 9.1 # C\n",
+    "mass_w = rho_w*tank_vol # kg\n",
+    "Q_cws = Q(mass_w, cp_w, dT)\n",
+    "print(\"The energy stored in the full CWS tank is {} joules\".format(Q_cws))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Converting Joules to MWh\n",
+    "\n",
+    "We can convert Joules to MWh by the following:\n",
+    "\n",
+    "$1 W = 1 J/s$ <br>\n",
+    "$1 Wh = 1 J/s * 3600 s/hour$<br>\n",
+    "$1 MWh = 1x10^6 Wh$\n",
+    "\n",
+    "So\n",
+    "\n",
+    "$ # MWh = # J / (1e6 * 3600)$"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The MWh equivalent in energy storage is 259.7423130260943 MWh\n"
+     ]
+    }
+   ],
+   "source": [
+    "Q_MWh = Q_cws / 3.6e9\n",
+    "\n",
+    "print(\"The MWh equivalent in energy storage is {} MWh (thermal)\".format(Q_MWh))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Conversion Efficiency\n",
+    "\n",
+    "Chilled water is a form of thermal energy storage. To get the equivalent energy storage in electric energy, a factor of 1/3 is multiplied as an assumption ($\\eta = 33\\%$)"
```

Comment:
What's the point of converting the energy from thermal to electricity? The thermal energy stored in the chilled water tank is never converted to electricity, right?

Reply:
As discussed in real life with @samgdotson, these are rough estimates. More accurate calculations require more accurate data.